src/pages/Pool.vue

```diff
@@ -110,6 +110,31 @@
           class="pool-actions-card"
         />
       </div>
+      <div class="order-1 lg:order-2 px-1 lg:px-0" v-else>
```

Comment:
Minor thing, but imo conditionals for rendering should always be the first attribute. It's probably the most important attribute in understanding how the template will render.
```suggestion
      <div v-else class="order-1 lg:order-2 px-1 lg:px-0">
```